cabal-install/Distribution/Client/IndexUtils.hs

```diff
@@ -262,7 +262,7 @@ getSourcePackagesAtIndexState verbosity repoCtxt mb_idxState = do
             return ()
         IndexStateTime ts0 -> do
             when (isiMaxTime isi /= ts0) $
-                warn verbosity ("Requested index-state " ++ display ts0
+                info verbosity ("Requested index-state " ++ display ts0
```

Comment:
OK. I was thinking of the case where the requested timestamp is invalid, e.g. in the future or deep past, so the command doesn't do what the user expects.